### PR TITLE
Add two fragments (greeting & info, quiz list) in the home activity

### PR DESCRIPTION
**Pull request by nbaehler**
I had no time to add tests up until now and I have to work on something else. I'll try to add some later today, but you can start reviewing.

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -105,6 +105,7 @@ dependencies {
     implementation 'com.google.firebase:firebase-analytics:17.2.3'
     implementation 'com.google.firebase:firebase-auth:19.3.0'
     implementation 'com.google.firebase:firebase-firestore:21.4.1'
+    implementation 'androidx.recyclerview:recyclerview:1.0.0'
```
Comment (nbaehler) [inline app/build.gradle]: I'll check if we need this one ...

File: app/src/main/res/values/dimens.xml [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <resources>
     <!-- Default screen margins, per the Android Design guidelines. -->
- <dimen name="activity_horizontal_margin">16dp</dimen>
- <dimen name="activity_vertical_margin">16dp</dimen>
+    <dimen name="activity_horizontal_margin">16dp</dimen>
+    <dimen name="activity_vertical_margin">16dp</dimen>
+    <dimen name="text_margin">16dp</dimen>
```
Comment (nbaehler) [inline app/src/main/res/values/dimens.xml]: I'll check this as well.

File: app/src/main/java/ch/epfl/qedit/backend/auth/AuthenticationFactory.java [hunk 3]
```diff
@@ -9,7 +9,7 @@
     private static AuthenticationService authService = null;
 
     public static AuthenticationService getInstance() {
-        if (authService == null) authService = new FirebaseAuthService();
+        if (authService == null) authService = new MockAuthService();
```
[the remark this answers is not in the document]
Comment (nathan96g) [reply app/src/main/java/ch/epfl/qedit/backend/auth/AuthenticationFactory.java]: oh okay, for demonstration purpose right ?

[the remark this answers is not in the document]
Comment (nbaehler) [reply app/src/main/java/ch/epfl/qedit/backend/auth/AuthenticationFactory.java]: Yes, and I could test manually without needing to copy/paste tokens.

[the remark this answers is not in the document]
Comment (nathan96g) [reply app/src/main/java/ch/epfl/qedit/backend/auth/AuthenticationFactory.java]: Okay for me you can merge whenever you think you have enough coverage. The functionnality seems to work.

**Pull request merged**